test/framework/dockerfile_provider.go

```diff
@@ -70,13 +85,16 @@ func GetKogitoApplicationDockerfileProvider(projectLocation string) KogitoApplic
 	return dockerfileProvider
 }
 
-func (dockerfileProvider *kogitoApplicationDockerfileProviderStruct) GetDockerfileContent() string {
+func (dockerfileProvider *kogitoApplicationDockerfileProviderStruct) GetDockerfileContent() (string, error) {
 	// Declare base image to build from
 	dockerfileContent := fmt.Sprintf("FROM %s\n", GetBuildImage(dockerfileProvider.imageName))
 
 	// Copy application binary into $KOGITO_HOME/bin
-	applicationName := filepath.Base(dockerfileProvider.projectLocation)
-	dockerfileContent += fmt.Sprintf("COPY target/%s%s $KOGITO_HOME/bin\n", applicationName, dockerfileProvider.applicationBinarySuffix)
+	applicationBinaryFileName, err := dockerfileProvider.getApplicationBinaryFileName(dockerfileProvider.projectLocation + "/target")
```

Comment:
That change was done because of Travel agency example. That example produces artifacts which contain also project version in their name. So I needed to introduce some generic approach to detect proper application binaries.

Reply:
all examples binaries should be without the version number in name.
missing the `build.finalName` in pom ... gonna PR to update that

Reply:
https://github.com/kiegroup/kogito-examples/pull/319

Reply:
Yeah, that's what I had in mind, thanks @radtriste!

Reply:
@sutaakar https://issues.redhat.com/browse/KOGITO-2970 has been done. should we update that part in BDD tests ?

Reply:
I guess it can be reverted back. Do you want to do it or should I do it?

Reply:
I can try it :)

Reply:
https://issues.redhat.com/browse/KOGITO-2988